nautobot_ssot/integrations/dna_center/constants.py

```diff
@@ -0,0 +1,33 @@
+"""Defines various constants used throughout the App."""
+
+from django.conf import settings
+
+# Import config vars from nautobot_config.py
+PLUGIN_CFG = settings.PLUGINS_CONFIG["nautobot_ssot"]
+
+DNAC_PLATFORM_MAPPER = {
```

Comment:
Can we add to netutils and use that here?

Reply:
Sure, I can look into getting this added to netutils. When I made this I didn't think it was something worth putting in netutils but I can see the value if we add it to the network_driver mapping.